src/components/ItemsList/ItemsList.tsx

```diff
@@ -0,0 +1,15 @@
+import { Item } from '../Item/Item';
+
+interface Props {
+  items: string[];
+}
+
+export const ItemList: React.FC<Props> = ({ items }) => {
+  return (
+    <ul>
+      {items.map((item, index) => (
+        <Item name={item} key={index} />
```

Comment:
Using the array index as a key in React is not recommended if the list can change because it can negatively impact performance and may cause issues with component state. If the `items` have unique values, you can use the item value as a key, otherwise consider using a unique ID.